### PR TITLE
chore: use `href` instead of deprecated to `prop` in InlineLink usage

### PR DESCRIPTION
**Pull request by enjoyooor**
Dupe of #12218 (accidentally closed), prefer to contribute using my personal account

Noticed that the `to` prop in `InlineLink` component is deprecated, so updating all usages to reflect this.

## Summary by CodeRabbit

- **New Features**
	- Enhanced navigation experience by updating internal link handling to standard web link behavior across various components and pages.

- **Bug Fixes**
	- Corrected link destinations by switching from custom `to` props to standard `href` attributes, ensuring proper redirection and improving user navigation on the site.

- **Refactor**
	- Streamlined linking mechanism throughout the application for consistency and improved maintainability.

- **Chores**
	- Removed unnecessary imports and updated the codebase to align with best practices for link handling.

### CODE REVIEW

**Comment by enjoyooor** [comment]
cc @nhsz would love your 👀

**Review by minimalsm — APPROVED** [review]
LGTM, thanks @enjoyooor!

Just waiting on second review as it touches a lot.

**Comment by minimalsm** [comment]
@all-contributors please add @enjoyooor for code

File: src/pages/dapps.tsx
Comment (pettinarip) [inline src/pages/dapps.tsx:113]: Why are we doing this?

**Review by pettinarip — CHANGES_REQUESTED** [review]
Thanks for the refactor @enjoyooor. Everything looks good but there is one import condition that I don't think is necessary and shouldn't be included in the scope of this PR. LMK what do you think.

File: src/pages/dapps.tsx
Comment (TylerAPfledderer) [reply src/pages/dapps.tsx:113]: That is indeed an auto-fix from eslint. Else the following error throws:
`Module '"*.png"' has no exported member 'uniswapec'. Did you mean to use 'import uniswapec from "*.png"' instead?`

I'm unfamiliar with how this is possible to get multiple named imports from a single image file anyway 🤔

Comment (TylerAPfledderer) [reply src/pages/dapps.tsx:113]: @pettinarip that is indeed an auto-fix from eslint, as it is the same image file imported twice. I guess eslint is unaware that it's an image file, so it consolidated the alias naming.

I'll look at throwing up a quick PR to address this duplication.

Comment (pettinarip) [reply src/pages/dapps.tsx:113]: Aha! didn't notice that, good call, thanks. Ok, so I guess we are ok then here 👍🏼

**Review by pettinarip — APPROVED** [review]
Thanks @enjoyooor! 💯

**Pull request merged**
